WALLET.md

```diff
@@ -62,8 +62,8 @@ Install Node version listed in [.nvmrc](.nvmrc) and make it default (example for
 
 ```bash
 # Check actual version number in .nvmrc
-nvm install 16.20.2
-nvm alias default 16.20.2
+nvm install 20
+nvm alias default 20
 ```
```

Comment:
The version number can be omitted. Also the alias can be set via `--default` option.
```md
nvm install [<version>]                     Download and install a <version>. Uses .nvmrc if available and version is omitted.
  --default                                 After installing, set default alias to the version specified. (same as: nvm alias default <version>)
```

```suggestion
nvm install --default
```